### PR TITLE
chore: Adds catalog-info.yml for the project

### PR DESCRIPTION
**Pull request by farhaanbukhsh**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @farhaanbukhsh! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by farhaanbukhsh** [comment]
@feanil @arbrandes  can you folks have a look?

File: catalog-info.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: "frontend-app-gradebook"
+  description: "The frontend (MFE) for Open edX Gradebook"
+  links:
+    - url: "https://github.com/openedx/frontend-app-gradebook"
+      title: "Frontend App Gradebook"
+      icon: "Web"
```
Comment (feanil) [inline catalog-info.yaml:11]: You don't need to provide a link to the code, that is done automatically in backstage.

**Review by feanil — APPROVED** [review]
One suggestion but then feel free to merge without another round of review.

File: catalog-info.yaml (hunk 1 above)
Comment (farhaanbukhsh) [reply catalog-info.yaml]: thanks

**Comment by openedx-webhooks** [comment]
@farhaanbukhsh 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-app-gradebook&entry.867055334=https://github.com/openedx/frontend-app-gradebook/pull/393&entry.1484655318=https://github.com/farhaanbukhsh&entry.752974735=2024-05-22+16:14&entry.1917517419=2024-05-23+07:57&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**